lib/cocoapods/target/pod_target.rb

```diff
@@ -447,6 +447,10 @@ def prefix_header_path
       support_files_dir + "#{label}-prefix.pch"
     end
 
+    def info_plist_entries
+      spec_consumers.first.info_plist
```

Comment:
should this be merging from different consumers or something?

Reply:
yeah I had this same question, I realized I'm not sure of how/when pod targets would have multiple specs, I assumed it was for either subspecs or scoped targets.

there was 1 or 2 other places that do `.first` like [here](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/target/pod_target.rb#L212). do you have some info on when there would be multiple specs for 1 target?

Reply:
A pod target can be assigned to multiple target definitions. The link you posted has already been validated during analysis to ensure that the target definitions the pod target belong to do not have different Swift versions set and if so it will error out.

I am not sure we want to arbitrarily return the first spec consumers info plist here. Do you support specifying info plist hashes within subspecs?

Reply:
Multiple target definitions shouldn't be an issue but yeah if one PodTarget instance represents all variants of a pod (ex. different subspecs in different target definitions) then yeah I'll need to change this.

Currently PodTargetInstaller generates an info.plist based on the PodTarget and not any individual specs (see [here](https://github.com/CocoaPods/CocoaPods/blob/45f1f22cf4b7e8663c34d490d8f5d9a93cb5b6f2/lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb#L91) so I'll have to change that as well

Reply:
Discussed this in the RFC a bit but for context: I realized subspec support doesn't quite make sense, so I've removed it from the DSL and will update this to always refer to the root